### PR TITLE
clean up headers

### PR DESCRIPTION
**Pull request by cheld**
Did some manual testing and grep searches.

1. Removed 2 double licenses
2. Replaced a couple of old Google licenses with current template
3. Removed unused favicon originally generated by yeoman

I noticed we are using Kubernetes Dashboard Autors. Other projects use Kubernetes Autors e.g. kops, minikube. Not sure if it matters. I think we should merge to make it consistent and then make a quick replace with all headers to be on the save side. I can do so.

### CODE REVIEW

**Comment by maciaszczykm** [comment]
> I noticed we are using Kubernetes Dashboard Autors. Other projects use Kubernetes Autors e.g. kops, minikube. Not sure if it matters. I think we should merge to make it consistent and then make a quick replace with all headers to be on the save side. I can do so.

+1

**Comment by maciaszczykm** [comment]
We should update Wiki footer too:

https://github.com/kubernetes/dashboard/wiki

File: build/assets/license/header_yaml.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 The Kubernetes Dashboard Authors.
```
Comment (floreks) [inline build/assets/license/header_yaml.txt]: We keep here license template files used by gulp tasks to automatically update file headers. You would need to update `check-license-headers` and `update-license-headers` gulp tasks. Right now it is not used.

Comment (cheld) [reply build/assets/license/header_yaml.txt]: Yea, I just needed a place for copy&paste manually a couple of files

Comment (cheld) [reply build/assets/license/header_yaml.txt]: Should the file be removed?

Comment (floreks) [reply build/assets/license/header_yaml.txt]: If it won't be used by our build pipeline then yes.

**Comment by cheld** [comment]
PTAL

**Comment by cheld** [comment]
Fixes https://github.com/kubernetes/dashboard/issues/2337

**Comment by cheld** [comment]
I thought I do it seperately. Find a way to automate, anounce, and then we do a quick merge. So we have less merge conflicts.

**Comment by maciaszczykm** [comment]
@cheld Sure.

**Pull request merged**